formulation/templatetags/formulation.py

```diff
@@ -118,7 +118,14 @@ def render(self, context):
 @register.simple_tag(takes_context=True)
 def field(context, field, widget=None, **kwargs):
     if isinstance(field, six.string_types):
-        field = context['formulation-form'][field]
+        formulation_form = context['formulation-form']
+        if formulation_form is None:
+            raise ValueError(
+                'The form field you\'re trying to access probably doesn\'t '
```

Comment:
This error message is incorrect.

This case is when you try to indirectly reference a field [using a string] but haven't specified a form in the form tag.